cmd/lint/main.go

```diff
@@ -39,7 +39,7 @@ var commandRules = []linter.CommandRule{
 	linter.RequireValidExamples(),
 
 	// Soft Requirements
-	linter.RequireLengthBetween("Short", 10, 60),
+	linter.RequireLengthBetween("Short", 10, 64),
```

Comment:
Instead of increasing this, can you just add the relevant command to the whitelist if you can't shorten it? 60 is arbitrary, but we don't want to get in the habit of changing it.